lib/internal/util/parse_args/parse_args.js

```diff
@@ -336,6 +337,16 @@ const parseArgs = (config = kEmptyObject) => {
     }
   });
 
+  // Phase 3: check if some options are required
+  ArrayPrototypeForEach(
+    ObjectEntries(options),
+    ({ 0: longOption, 1: optionConfig }) => {
+      if (optionConfig.required && result.values[longOption] === undefined) {
```

Comment:
The `options` came from the user, and is subject to possible prototype pollution. There is a helper routine `objectGetOwn` to do this more robustly:

https://github.com/nodejs/node/blob/e62f6ce630c3ea9d9484d14bb892cb6836b250d4/lib/internal/util/parse_args/utils.js#L26

(The `result` was made internally with null prototype so can be used more conventionally with safety.)